### ISSUE TITLE
Tooltip gets accessibility violations

### ISSUE THREAD

**Issue by yrue** [carbon-design-system/ibm-security#881]
## Bug - 
Click icon to show tooltip and an accessibility violation 'Content is not within a landmark element' shows.

#### Expected behavior

No violation

#### Actual behavior

There's accessibility violation now.

### Steps for reproducing

1. go to storybook https://react.carbondesignsystem.com/?path=/story/tooltip--default-bottom
2. use component tool to force show the tooltip (set prop `open: true` as below screenshot) 
![image](https://user-images.githubusercontent.com/5738032/102735505-9d105b80-437d-11eb-9ee1-dc517fadedf8.png)
3. (Chrome) go to Elements Tab and open Accessibility Checker and run
4. observed
![image](https://user-images.githubusercontent.com/5738032/102735754-33448180-437e-11eb-99cf-f123a1eaa022.png)
![image](https://user-images.githubusercontent.com/5738032/102735782-3e97ad00-437e-11eb-9bed-e1e034cb306d.png)

### Affected browsers

[What's my browser?](http://www.whatsmyua.com) and [browserl.ist supported browsers](https://github.com/carbon-design-system/ibm-security/blob/master/package.json#L174-L178)

- [x] Chrome 50 on Mac OSX 10
- [ ]  / 
- [ ]  / 

### Optional information

**Version -** 1.36.1-prerelease.2

**Comment by SimonFinney** [comment]
Related to https://github.com/carbon-design-system/carbon/issues/6308
